src/components/PopupDelete.js

```diff
@@ -0,0 +1,28 @@
+import Popup from "./Popup";
+
+export class PopupDelete extends Popup {
+  constructor({ popupSelector }, handleDelete) {
+    super({ popupSelector });
+    this._popupDelete = document.querySelector("#delete__card-modal");
+    this._deleteButton = document.querySelector("#delete__card-button");
+    this._handleSubmit = handleDelete;
+    
+  }
+
+  setSubmitAction(action) {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.

Reply:
thank you so much!